chimerapy/manager.py

```diff
@@ -172,6 +173,10 @@ async def update_nodes_status(self, request: web.Request):
         logger.debug(f"{self}: Nodes status update to: {self.state.workers}")
 
         # Relay information to front-end
+        await self.server.async_broadcast(
+            signal=MANAGER_MESSAGE.NODE_STATUS_UPDATE,
+            data=self.state.to_dict(),
+        )
```

Comment:
After https://github.com/oele-isis-vanderbilt/ChimeraPy/pull/115, is merged, I think this belongs to `API`?

Reply:
That is a great point. Okay, will add commit to refactor the code into ``API``.

Reply:
Added to the latest commit 36802b1